src/routes/patterns/components/Buttons.js

```diff
@@ -1,28 +1,72 @@
-import React from 'react';
-
-import { Button, ButtonGroup, Card } from 'styled';
+import React from "react";
+import { Button, ButtonGroup, Card, CodeBlock, Subheading } from "styled";
 
 const Buttons = () => (
-  <Card className='Buttons'>
-    <h4 className='legend'>Buttons</h4>
+  <Card mb={24} padded>
+    <Subheading mb={24}>Buttons</Subheading>
+
+    <Subheading capitalize color micro mb={8}>Button</Subheading>
     <ButtonGroup>
-      <Button>Default Button</Button>
```

Comment:
Buttons should have a :focus and :active state for accessibility ideally. Perhaps we can address this in the future.

Reply:
@hudakdidit added these to the mix 👍